Shared/SBTerminals.swift

```diff
@@ -68,6 +68,9 @@ extension SBObject: SBiTermItem {}
     @objc optional var uriToken: String { get } // URI token
 
     @objc optional func classForScriptingClass(_ className: String) -> AnyClass?
+    
+    
+
```

Comment:
Why?

Reply:
On building, these errors appear:

```
ScriptingBridge.SBApplication:17:15: error: Objective-C method 'classForScriptingClass:' provided by method 'class(forScriptingClass:)' conflicts with optional requirement method 'classForScriptingClass' in protocol 'SBiTermITermApplication'
    open func `class`(forScriptingClass className: String) -> Swift.AnyClass?
              ^
ScriptingBridge.SBApplication:17:15: note: rename method to match requirement 'classForScriptingClass'
    open func `class`(forScriptingClass className: String) -> Swift.AnyClass?
              ^
ScriptingBridge.SBApplication:17:15: note: add '@nonobjc' to silence this error
    open func `class`(forScriptingClass className: String) -> Swift.AnyClass?

```

Attempting conversion.
Ideally we should get rid of ScriptingBridge altogether as the AppStore no longer supports it.